docs/k8s_example.yml

```diff
@@ -85,6 +85,19 @@ spec:
     role: leader
     tier: backend
 
+---
+apiVersion: "v1"
+kind: "ConfigMap"
+metadata:
+  name: "custom"
+  namespace: "ots"
```

Comment:
Lets maybe remove the `namespace` metadata completely… When executed using `kubectl -n myots create -f ...` this ignores the `myots` namespace and creates an `ots` namespace… Without specifying a namespace it should respect the `-n` flag if I'm not fully mistaken…

Reply:
Removed in 2a5f45dbd4c255d75c9c4119dc141d40b9e3de5b.

Reply:
Please also remove the first doc creating the namespace

Reply:
Just drop everything after and including the first dot… `tcp://ots-redis:port`

Reply:
~Doesn't that default to the `default` namespace?~ Oh, you're right. It works by setting a search domain in every pod, I thought they expanded it in the DNS service.